gradlew

```diff
@@ -32,10 +32,10 @@
 #       Busybox and similar reduced shells will NOT work, because this script
 #       requires all of these POSIX shell features:
 #         * functions;
-#         * expansions «$var», «${var}», «${var:-default}», «${var+SET}»,
-#           «${var#prefix}», «${var%suffix}», and «$( cmd )»;
-#         * compound commands having a testable exit status, especially «case»;
-#         * various built-in commands including «command», «set», and «ulimit».
+#         * expansions �$var�, �${var}�, �${var:-default}�, �${var+SET}�,
+#           �${var#prefix}�, �${var%suffix}�, and �$( cmd )�;
+#         * compound commands having a testable exit status, especially �case�;
+#         * various built-in commands including �command�, �set�, and �ulimit�.
```

Comment:
Something weird happened here. When I run `./gradlew wrapper on my computer, it generates the following diff:

![image](https://user-images.githubusercontent.com/24747/220931022-f29cda04-957b-4f91-adea-05a0946415d4.png)

Reply:
If you can fix this, I think we can merge this PR.

Reply:
I pushed an additional commit to fix this.

Reply:
From the top of my head this may be related with zero width characters or something od that sort (https://github.com/microsoft/terminal/issues/11850) 

Thing is that I am constantly switching from Windows to Linux and back, and this commit was created in Windows.

In any case, thank you for solving this.